docs/python-client.rst

```diff
@@ -50,16 +63,27 @@ you wish to connect to. The easiest way to do so is to pass the full URL to the
 REST API of the Girder instance you wish to connect to using the ``api-url``
 argument to ``girder-cli``. For example: ::
 
-    girder-cli --api-url http://localhost:8080/api/v1 <subcommand> ...
+    girder-cli --api-url http://localhost:8080/api/v1 <command> ...
 
-You may also specify the URL in parts, using the ``host`` argument, and optional
-``scheme``, ``port``, and ``api-root`` args. ::
+For backward compatibility, you may also specify the URL in parts, using the ``host`` argument,
```

Comment:
This isn't really related to backward compatibility, just a matter of preference.